Makefile

```diff
@@ -187,18 +190,23 @@ GINKGO ?= $(LOCALBIN)/ginkgo
 ## Tool Versions
 KUSTOMIZE_VERSION ?= v3.8.7
 CONTROLLER_TOOLS_VERSION ?= v0.11.1
+OPERATOR_SDK_VERSION ?= v1.31.0
```

Comment:
This is now  a duplicate with L51

Reply:
good point I caught it in another review missed it here

Reply:
done